pureStorageDriver/README.md

```diff
@@ -262,35 +261,10 @@ NAME     STATUS   READY   RANGES   UNDER-REPLICATED   UNAVAILABLE   AS-OF
 pso-db   Live     3/3     31       0                  0             2020-03-05T00:40:38Z
 ```
 
-## How to update `arrays` info
-
-Update your values.yaml with the correct arrays info, and then upgrade the helm as below.
-
-**Note**: Ensure that the values for `--set` options match when run with the original install step. It is highly recommended
-to use the values.yaml and not specify options with `--set` to make this easier.
-
-```bash
-helm upgrade pure-storage-driver pure/pureStorageDriver --version <version> --namespace <namespace> -f <your_own_dir>/yourvalues.yaml --set ...
-```
-
 ## Uninstall
 
-To uninstall, first delete `pure-provisioner-secret`
-
-```bash
-kubectl delete secret -n <namespace> pure-provisioner-secret
-```
-
-Once the secret is deleted, `pso-db-cockroach-operator` will begin detaching the database volumes from the compute nodes.
-
-Verify that all of the database volumes were cleaned up by running:
-```bash
-kubectl get intrusion -n <namespace>
-```
-If no intrusion is returned, it is now safe to helm uninstall:
-```bash
-helm uninstall pure-storage-driver --namespace <namespace>
-```
+To uninstall, run `helm delete -n <namespace> pure-storage-driver`. Most resources will be immediately removed, but
```

Comment:
nit: may follow previous instructions, put a bash code block here, clear and easy to copy.